randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -552,14 +552,17 @@ Extra - room_id: [13]
               # Core-X Requirements
               Missiles
               Combat (Beginner) or 20 DMG Missiles
+              # A way to dodge
+              Combat (Intermediate) or Have Any Jump Upgrade
```

Comment:
is this a thing for every core-x? or is this just for mega core?

Reply:
this is only for the core-x of mega-core. It's very fast and if people don't know how to dodge it (which isnt exactly obvious), will get hit.